viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts

```diff
@@ -148,6 +151,10 @@ export class FlexibleControls extends EventDispatcher<FlexibleControlsEvent> {
     return this._camera;
   }
 
+  get domElement(): HTMLElement {
```

Comment:
Why controls should expose `DOM Element`, isn't `viewer._domElement` and this same?

Reply:
I don' have the viewer here